draft/index.html

```diff
@@ -794,60 +799,17 @@
     <dd>keine</dd>
     <dt>Elemente:</dt>
     <dd>
-        <a>&lt;source></a> mit dem Wert `Hochschulcampus Content Custom Model`<br>
-        <a>&lt;value></a>
-
-        Für das `Hochschulcampus Content Custom Model` sind die erlaubten Werte (mit
-        erläuternden Bedeutungen in deutsch), die bei Bedarf und nach Abstimmung
-        erweitert werden können:
-
-        - `application` - Anwendung, Software
-
-        - `assessment` - Lernkontrolle, (Selbst-)Test
-
-        - `audio` - Tonaufnahme
-
-        - `case_study` - Fallstudie
-
-        - `course` - Lernkurs
-
-        - `data` - (Roh-, Beispiel-) Daten
-
-        - `diagram` - Diagramm, Grafik
-
-        - `drill_and_practice` - Übung
-
-        - `educational_game` - Lernspiel
-
-        - `experiment` - Experiment
-
-        - `image` - Abbildung, Foto, Bild
-
-        - `index` - Glossar, Nachschlagewerk, Wiki, Lexikon
-
-        - `lesson_plan` - Unterrichtsplanung, Unterrichtsgestaltung
-
-        - `map` - Karte
-
-        - `portal` - (Web-, Wissens-) Portal
-
-        - `questionnaire` - Fragebogen, Rechercheauftrag
-
-        - `script` - Skript
-
-        - `simulation` - Simulation
-
-        - `slide` - Präsentation, Folien
-
-        - `text` - Artikel, Aufstatz, Text, Abhandlung 
-
-        - `video` - Video
-
-        - `web_page` - Webseite
-
-        - `worksheet` - Arbeitsblatt, Arbeitsmaterial
+        <a>&lt;source></a> mit dem Wert `https://w3id.org/dini-ag-kim/hcrt/scheme`<br>
+        <a>&lt;id></a><br>
+        <a>&lt;entry></a> - Es benennt die konkrete Art des Objekts
+        mittels eines Labels. Da die <a>&lt;id></a>
+        das entscheidende Klassifikationsmerkmal ist, hat dieses Element nur
+        beschreibenden Charakter.<br>
 
-        - `other` - sonstiges
+        Für die Arten des Lernobjekts wurden „Hochschulcampus Ressourcentypen“
+        in Anlehnung an LOM spezifiziert, die in [[!SKOS-LRT]] repräsentiert
+        sind. Die angegebene URL in <a>&lt;source></a> stellt den persistenten
+        Link zur Repräsentation dar.
```

Comment:
Ich finde insbesondere den letzten Satz unklar. Worauf bezieht sich da "Repräsentation"? M.E. kann der Satz einfach weggelassen werden. Formulierungsvorschlag:

```suggestion
        Für die möglichen Werte wurde mit „Hochschulcampus Ressourcentypen“ 
        ein kontrolliertes Vokabular in Anlehnung an LOM spezifiziert ([[!SKOS-LRT]]), 
        in dem jeder mögliche Wert durch einen URI identifiziert ist.
```